config.example.yml

```diff
@@ -5,13 +5,25 @@ server:
   adminConnectors:
   - type: http
     port: ${MARQUEZ_ADMIN_PORT:-8081}
+#  requestLog:
+#    appenders:
+#      - type: console
+#        layout:
+#          type: access-json
 
 db:
   driverClass: org.postgresql.Driver
   url: jdbc:postgresql://${POSTGRES_HOST:-localhost}:${POSTGRES_PORT:-5432}/${POSTGRES_DB}
   user: ${POSTGRES_USER}
   password: ${POSTGRES_PASSWORD}
 
+logging:
+  level: ${LOG_LEVEL:-INFO}
+  appenders:
+    - type: console
+#      layout:
```

Comment:
should this be commented out?

Reply:
Yeah, it's more for illustrating an example configuration of JSON logging